cosmic-applet-status-area/src/components/app.rs

```diff
@@ -26,6 +26,7 @@ pub enum Msg {
     TogglePopup(usize),
     Hovered(usize),
     Surface(surface::Action),
+    RightPressed(usize),
```

Comment:
I'm not sure what to name this.
If we go with this name, it might also make sense to rename `TogglePopup`  to `LeftPressed` to better reflect the logic.

Reply:
It should be named based on what action the message will perform.

Reply:
Thank you, I renamed it to `OpenContextMenu`. Let me know if that's a suitable name.